edx_django_utils/user/__init__.py

```diff
@@ -4,17 +4,96 @@
 
 import random
 import string
+from django.conf import settings
+
+non_ascii_characters = [
```

Comment:
Just a passing comment, but it seems odd that this is a fixed list of characters. Where did this come from? Can it be commented?

Reply:
That were a static list of symbols that were used if the password policy validators include at least 1 symbol (`common.djangoapps.util.password_policy_validators.SymbolValidator`) with a `min_symbol` option.

Viewing your comment, I decide to replace that static list with a function that iterates all chars and use all symbols. The impact is a small slower performance.
```python
def _symbols():
    """Get all valid symbols"""
    symbols = []
    for c in map(chr, range(sys.maxunicode + 1)):
        if 'S' in unicodedata.category(c):
            symbols.append(c)
    return symbols
```